chore: Make cicd run

marc-niclas opened the pull request.

marc-niclas committed:
chore: Make cicd run

marc-niclas committed:
add readme

marc-niclas committed:
feat: Trigger first release

semantic-release-bot committed:
chore(release): 1.0.0 [skip ci]

## 1.0.0 (2025-03-18)

### Features

* Trigger first release ([82cd551](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/commit/82cd55193254997d0390e7c361d91d107ba40ed3))

bne-admin commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/ts-ecma-import-codmod/v/1.0.0)
- [GitHub release](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

marc-niclas committed:
fix: Try to get version command from package JSON

marc-niclas committed:
Merge branch 'get-cicd-running' of github.com-work:BrightNight-Energy/ts-ecma-import-codmod into get-cicd-running

marc-niclas committed:
removed tsx

marc-niclas committed:
formatting

semantic-release-bot committed:
chore(release): 1.0.1 [skip ci]

## [1.0.1](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/compare/v1.0.0...v1.0.1) (2025-03-18)

### Bug Fixes

* Try to get version command from package JSON ([8a3ada9](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/commit/8a3ada905d7abc73a77b4fdd57e5bdcb5cc6d5f9))

bne-admin commented:
:tada: This PR is included in version 1.0.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/ts-ecma-import-codmod/v/1.0.1)
- [GitHub release](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/releases/tag/v1.0.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

marc-niclas committed:
fix: Fix bin script

marc-niclas committed:
Merge branch 'get-cicd-running' of github.com-work:BrightNight-Energy/ts-ecma-import-codmod into get-cicd-running

semantic-release-bot committed:
chore(release): 1.0.2 [skip ci]

## [1.0.2](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/compare/v1.0.1...v1.0.2) (2025-03-18)

### Bug Fixes

* Fix bin script ([a19eccf](https://github.com/BrightNight-Energy/ts-ecma-import-codmod/commit/a19eccf3e864956b130c999d4c0cdcd2498eb3ef))

marc-niclas committed:
update readme

marc-niclas committed:
Merge branch 'get-cicd-running' of github.com-work:BrightNight-Energy/ts-ecma-import-codmod into get-cicd-running

marc-niclas committed:
restore cicd
